include/core/size.hpp

```
@@ -72,3 +84,90 @@ inline constexpr UISize operator/(const UISize &lhs, float rhs) noexcept {
+
+constexpr UISizing operator-(const UISizing &lhs, const UISizing &rhs) noexcept {
+  return UISizing{lhs.width - rhs.width, lhs.height - rhs.height, lhs.typeW, lhs.typeH};
+}
+
+constexpr UISizing operator*(const UISizing &lhs, float rhs) noexcept {
+  return UISizing{lhs.width * rhs, lhs.height * rhs, lhs.typeW, lhs.typeH};
+}
+
+constexpr UISizing operator*(float lhs, const UISizing &rhs) noexcept { return rhs * lhs; }
+
+constexpr UISizing operator/(const UISizing &lhs, float rhs) noexcept {
```

Clang (14.0.6)

[warning] use a trailing return type for this function

(modernize-use-trailing-return-type)